cwltool/main.py

```diff
@@ -630,7 +631,8 @@ def main(argsl=None,  # type: List[str]
                 setattr(args, "workflow", "CWLFile")
             else:
                 _logger.error("")
-                _logger.error("CWL document required, try --help for details")
+                _logger.error("CWL document required, No input file was passed\n")
```

Comment:
Is the `\n` really needed?

I would say "CWL document required, no input file was provided"